### PR TITLE
some housekeeping

### PR DESCRIPTION
**Pull request by attila-lendvai**
mostly just rewordings, but

- fixes the Bzzaar URL's status=%d
- turns an error log into a warning

### CODE REVIEW

File: cmd/bee/cmd/start.go
Comment (acud) [inline cmd/bee/cmd/start.go:257]: I think the wording on the left is better

**Review by acud — APPROVED** [review]
Minor comments. Otherwise good to go

File: cmd/bee/cmd/start.go
Comment (attila-lendvai) [reply cmd/bee/cmd/start.go:257]: @acud "on the left"? you mean the original? my problem with the original is that it doesn't communicate that it's an ongoing process, and that the failure is temporary. hence my issue with the use of `Errorf`, and the original wording that suggests that it's something final.

note that this change was motivated by an actual confusion of mine, that lasted until i looked at the code.

Comment (attila-lendvai) [reply cmd/bee/cmd/start.go:257]: an alternative: `trying to connect to clef signer, but failing due to error %v`

Comment (acud) [reply cmd/bee/cmd/start.go:257]: This is not really in the English standard-library :) trying and failing at the same time? failed since the failure is already known therefore in past tense. Sorry for the grammar harassment.  

> "on the left"? you mean the original

yes

**Review by zelig — APPROVED** [review]

File: cmd/bee/cmd/start.go
Comment (attila-lendvai) [reply cmd/bee/cmd/start.go:257]: the grammar of spoken languages is not set in stone... ;) i don't care how, but what i pointed out should be expressed in a way that is not confusing. the original wording suggests a final, failed situation.

i still think it's clear english: the trying is still happening, continuously, but it also keeps failing, for now. but it's rather funny when two non-native speakers get into such discussions... :)

**Comment by attila-lendvai** [comment]
a question: who is supposed to press the merge button?

1. the PR opener, once it has been enabled by the two approvals?
2. or shall i just leave it to the regular devs, who go through the PRs every once in a while?

File: cmd/bee/cmd/start.go
Comment (acud) [reply cmd/bee/cmd/start.go:257]: right. let's ask a native then. @significance?

Comment (Eknir) [reply cmd/bee/cmd/start.go:257]: No strong opinion here. I would like to ask @significance to cut the ties here.

**Review by Eknir — APPROVED** [review]
lgtm! Thanks @attila-lendvai . For the grammar issue; no strong opinion here. I think it was good as it was, but I am also not against your change.

File: cmd/bee/cmd/start.go
Comment (significance) [reply cmd/bee/cmd/start.go:257]: i guess maybe @attila-lendvai's rephrased alternative is the most proper? `trying to connect to clef signer, but failing due to error %v` but i think any would work for the use case presuming they are appearing multiple times in the logs?

personal vote goes to @acud's lolzcatz version 😹

Comment (attila-lendvai) [reply cmd/bee/cmd/start.go:257]: FTR, it usually appears only once in the log, because at the first retry clef is already ready and accepting connections.

**Pull request merged**
